doc/zip/README

```diff
@@ -1,10 +1,44 @@
-HTML:
+FromThePage Export Formats
 
-full -- full text of the transcription or translation, with versions, contributors, and comments.
-text -- just the full text of the transcription or translation.
+This archive contains files exported from FromThePage in one or more of the following formats:
 
-Text:
+HTML FORMATS:
 
-verbatim -- full text of the transcription, with versions, contributors and comments.
-searchable -- links removed and line breaks removed so searches can span line breaks.
-expanded -- expanded transcripts and translations replace the verbatim linked subjects with the canonical name of the subject to make it suitable for text analysis.
+full -- Full text of the transcription or translation, with versions, contributors, and comments.
+text -- Just the full text of the transcription or translation.
+transcript -- HTML transcript files only.
+translation -- HTML translation files only (if work supports translation).
+
+PLAINTEXT FORMATS:
+
+verbatim -- Full text of the transcription, with versions, contributors and comments. Line breaks are represented with single newlines, paragraph breaks with double newlines, and page breaks with triple newlines.
+searchable -- Text optimized for full-text search. Contains verbatim plaintext except that words broken by hyphenated newlines are joined together, and a list of canonical names mentioned within each page is appended.
+expanded -- Expanded transcripts and translations where linked subjects are replaced with their canonical names to make the text suitable for programmatic analysis (e.g., "Mr. Jones" becomes "James Jones").
+
+TEI XML FORMATS:
+
+tei -- Text Encoding Initiative XML format useful for editors who plan to do further mark-up within TEI-XML editors like oXygen.
```

Comment:
TEI-XML exports follow the TEI Consortium Guidelines and attempt to preserve all edit history, notes, mark-up, annotations and other features of the editorial process. It is the only "lossless" FromThePage export format.